tests/integration/golang/fixtures/run.go

```diff
@@ -238,3 +249,8 @@ func (f RunFixtures) CreateParams(
 	}
 	return nil
 }
+
+// needsContextAssociation indicates if the context FK is present.
+func needsContextAssociation(m *models.Metric) bool {
+	return m.ContextID == 0
+}
```

Comment:
we use this function as I can see just in one place. do we really need it? why we can't just use inline `m.ContextID == 0`?

Reply:
@dave-gantenbein requested a semantic test instead of `contextID == 0`

Reply:
emm, @dave-gantenbein ? I really can't see the advantage of function instead of `m.ContextID == 0`